From 178f7b1823806687cf4a64fb38ac014b4ed0801b Mon Sep 17 00:00:00 2001
From: Lasse Blaauwbroek <lasse@blaauwbroek.eu>
Date: Wed, 22 Jun 2022 13:14:35 +0200
Subject: [PATCH] Patch to inject Tactician into the standard library

---
 theories/Init/Ltac.v | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/theories/Init/Ltac.v b/theories/Init/Ltac.v
index fbd60a940f..ee0dd7e88d 100644
--- a/theories/Init/Ltac.v
+++ b/theories/Init/Ltac.v
@@ -9,5 +9,5 @@
 (************************************************************************)
 
 Declare ML Module "ltac_plugin:coq-core.plugins.ltac".
-
-Export Set Default Proof Mode "Classic".
+From Tactician Require Export Ltac1.Record.
+Export Set Default Proof Mode "Tactician Ltac1".
-- 
2.38.1

