From a34c1cc260daa35946eb9aba825ea783657a4b87 Mon Sep 17 00:00:00 2001
From: Lasse Blaauwbroek <lasse@blaauwbroek.eu>
Date: Thu, 23 Jun 2022 05:24:16 +0200
Subject: [PATCH] Patch to inject tactician into coq-hott

---
 theories/Basics/Notations.v | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/theories/Basics/Notations.v b/theories/Basics/Notations.v
index 279be582..eb3d76b9 100644
--- a/theories/Basics/Notations.v
+++ b/theories/Basics/Notations.v
@@ -10,7 +10,8 @@ Local Set Warnings Append "-skip-spaces-curly".
 
 (** ML Tactic Notations *)
 Declare ML Module "ltac_plugin".
-Global Set Default Proof Mode "Classic".
+From Tactician Require Export Ltac1.Record.
+Global Set Default Proof Mode "Tactician Ltac1".
 
 (** These are the notations whose level and associativity are imposed by Coq *)
 
-- 
2.36.1

